verdaccio/load-tests/scripts/soak-test.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Soak test with the purpose of testing an average load over a long period of time.
+ */
+
+import { sleep } from 'k6';
+import { requestTarball } from './common.js';
+
+const RAMP_DURATION = '5m';
+const STABLE_DURATION = '8h';
+const TARGET = 200;
```

Comment:
In the load test this was called `USERS_COUNT` which I find more informative. Can we use that terminology in all the tests?